frontend/src/bundles/common/components/base-layout/styles.module.css

```diff
@@ -3,7 +3,7 @@
         "header header"
         "aside main";
 
-    @apply grid grid-cols-[256px,auto] grid-rows-[88px,auto] overflow-hidden transition-all duration-[0.5s] ease-[ease-in-out] lg:grid-cols-[288px,auto];
+    @apply grid grid-cols-[0,auto] grid-rows-[88px,auto] overflow-hidden transition-all duration-[0.5s] ease-[ease-in-out] sm:grid-cols-[0,auto] md:grid-cols-[44px,auto]  lg:grid-cols-[288px,auto];
```

Comment:
Please, use `rem` everywhere in your PR instead of `px`

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/f11cc7af-a513-40b4-83bf-dcef38d933ca)
It tryed but it seems its not allowed